src/main/webapp/WEB-INF/tags/module/ar/invoiceGeneral.tag

```diff
@@ -0,0 +1,273 @@
+<%@ include file="/jsp/sys/kfsTldHeader.jsp"%>
+<%@ attribute name="readOnly" required="true" description="If document is in read only mode"%>
+
```

Comment:
The KualiCo version of "invoiceGeneral.tag" has a license header at the top of it. You may want to include that header in the overlay as well.